documentation/slate/source/examples/jsFiddle/journeys/demo.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * The following example aims at illustrating Navitia’s journey feature.
+ * It requests an itinerary from a cool spot (our headquarter in Paris)
+ * to another cool spot (the Eiffel Tower, Paris).
+ * Please note that the transportation data that is used for this sandbox
+ * is only an example for demo purposes an does not reflect
+ * the actual transportation supply on the Parisian area.
+ */
+
+$.ajax({
+  type: 'GET',
+  url: 'https://api.navitia.io/v1/journeys?from=2.3749036;48.8467927&to=2.2922926;48.8583736',
+  dataType: 'json',
+  headers: {
+    'Authorization': 'Basic ' + btoa('a61f4cd8-3007-4557-9f4c-214969369cca')
```

Comment:
Yep, maybe you can use the dummy token 123456789abcdef blablabla that is already mentionned in doc